dspace-spring-rest/src/test/java/org/dspace/app/rest/builder/CollectionBuilder.java

```diff
@@ -78,6 +79,26 @@ public Collection build() {
         return collection;
     }
 
+    /**
+    * Creates a TemplateItem for a given collection.
+    * 
+    * @param context DSpace Context
+    * @param collection DSpace Collection
+    */
+    public static Item createTemplateItem(final Context context, final Collection collection) {
```

Comment:
I'm not sure that we want to allow the use of a builder to modify the collection after the initial build. To be consistent with the existent code I would expect a withTemplateItem() method instead.

Reply:
I agree we should be consistent all around on how the code is implemented.